try{
  if(0){

  }
  else{
    throw "myException";
  }
}
catch (e){
  console.log(e);
}